Update Upgrade Documentation for all Installation Methods (#1733)
CrunchyData/postgres-operator: docs/content/Upgrade/automatedupgrade.md

##### NOTE: As with any upgrade procedure, it is strongly recommended that a full logical backup is taken before any upgrade procedure is started. Please see the [Logical Backups](/pgo-client/common-tasks#logical-backups-pg_dump--pg_dumpall) section of the Common Tasks page for more information.

> ### Automated Upgrade when using an Ansible installation, the PostgreSQL Operator Installer or the Helm Chart Installation of the PostgreSQL Operator

Reviewer: ```suggestion
### Automated Upgrade when using the PostgreSQL Operator Installer (`pgo-deployer`), Helm, or Ansible Ansible
```

Author: updated, thanks

#### Ansible

For existing PostgreSQL Operator deployments that were installed using Ansible, you will first need to check out the appropriate release tag of the Operator. Then please follow the [Update Instructions]({{< relref "installation/other/ansible/updating-operator.md" >}}), being sure to update the new inventory file with your required settings.

#### PostgreSQL Operator Installer

For existing PostgreSQL Operator deployments that were installed using the PostgreSQL Operator Installer, you will check out the appropriate release tag and update your the new configuration files. After this, you will need to update your Operator installation using the `DEPLOY_ACTION` method described in the [Configuring to Update and Uninstall](/installation/postgres-operator#configuring-to-update-and-uninstall) section of the documentation.

Please note, you will need to ensure that you have executed the [post-installation cleanup](/installation/postgres-operator#post-installation) between each `DEPLOY_ACTION` activity.

> #### Helm

Reviewer: the order of these sections should be 

- `#### PostgreSQL Operator Installer`
- `#### Helm`
- `#### Ansible`

Author: updated.
